### ISSUE TITLE
Update the edirect image

### ISSUE THREAD

**Issue by apcamargo** [ncbi/docker#36]
The Edirect image is very outdated by now. Are there any plans to update the image?

**Comment by christiam** [comment]
Yes, the image has been updated: https://hub.docker.com/r/ncbi/edirect/tags

**Comment by christiam** [comment]
Please let us know if you find any issues with it.

**Comment by apcamargo** [comment]
It seems to be working fine. Thank you!
